model/src/ww3_prnc.F90

```diff
@@ -798,10 +801,22 @@ PROGRAM W3PRNC
   IRET=NF90_GET_ATT(NCID,VARIDTMP,"calendar",CALENDAR)
   IF ( IRET/=NF90_NOERR ) THEN
     WRITE(NDSE,1028)
-  ELSE IF ((INDEX(CALENDAR, "standard").EQ.0) .AND. &
-       (INDEX(CALENDAR, "gregorian").EQ.0)) THEN
-    WRITE(NDSE,1029)
+  ELSE IF ((INDEX(CALENDAR, "standard") .GT. 0) .OR. &
+        (INDEX(CALENDAR, "gregorian") .GT. 0)) THEN
+    CALENDAR = "standard"
+  ELSE IF (INDEX(CALENDAR, "360_day") .GT. 0) THEN
+    CALENDAR = "360_day"
+  ELSE
+    WRITE(NDSE,1029) CALENDAR
+    CALENDAR = "standard" ! Default to standard calendar. Better to maybe fail?
```

Comment:
I'm also hesitating... If it fails, the error message should provide the following commandline to correct the missing calendar attribute :
ncatted -h -a calendar,time,c,c,'standard' file.nc

Reply:
I think I'd prefer to make it a failure to avoid unexpected results, if you agree?

Reply:
ok, it makes sense, could you add the nco command to help users to easily solve the issue ?

Reply:
Actually - that line is the error for catching a calendar type that is not currently handled by WW3 (e.g. `calendar='no_leap'`), so we would not want the user to change it to a standard calendar (as the times would be wrong). In this case i think it is right to fail.

The case where the calendar is not set at all still defaults to "standard" with a warning.

Reply:
@ukmo-ccbunney @mickaelaccensi, I wanted to check if this has been resolved to both your satisfaction?

Reply:
Hi @ukmo-ccbunney and @mickaelaccensi, I wanted to double check this conversation has been resolved before moving forward.  Is that the case?

Reply:
I am just waiting to hear from @mickaelaccensi to be sure he is happy with my reply above.

Reply:
@ukmo-ccbunney I agree with your last comment.